### PR TITLE
Add additional features to standings

### PR DESCRIPTION
**Pull request by ruairigibney**
- new aliases for commands
- new aliases for championships
- fix championship font
- fix text placement on the image
- derive current round from Rounds worksheets and display
- add new 'all' param to have bot return all championship standings
  (mods only)

### CODE REVIEW

File: TTBot/Models/ExcelSheetMappingModel.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace TTBot.Models
+{
+    class ExcelSheetMappingModel
+    {
+        public static Dictionary<string, string> championshipsMappings = new Dictionary<string, string>()
+        {
+            {"ACC ProAmChampionship", "ACC GT3 8pm"},
```
Comment (alex9158) [inline TTBot/Models/ExcelSheetMappingModel.cs:11]: I'd like to move this config to the DB at some point if possible, so we can avoid code changes when theres new championships.

File: TTBot/Program.cs [hunk 2]
```diff
@@ -275,7 +275,16 @@ private void ScaffoldDatabase()
                                     AS
                                     SELECT *, (select count(*) from EventSignup where EventId = event.Id) as ParticipantCount
                                     FROM [Event]
-                                    ");
+                                    ");
+
+                try
+                {
+                    connection.Execute(@"ALTER TABLE Event
```
Comment (alex9158) [inline TTBot/Program.cs:282]: Can you remove this and add as a script in the /scripts/ folder please?

File: TTBot/Models/ChampionshipAliasesModel.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace TTBot.Models
+{
+    public class ChampionshipAliasesModel
+    {
+        private static Dictionary<string, string[]> aliasesDictionary = new Dictionary<string, string[]>() {
+            { "ACC GT3 8pm", new string[]{ "ACC Pro", "Pro Am", "Pro/Am", "GT Pro", "Mon Pro", "Monday Pro" } },
```
Comment (alex9158) [inline TTBot/Models/ChampionshipAliasesModel.cs:12]: I'd like to move this config to the DB at some point if possible, so we can avoid code changes when theres new championships.

**Review by alex9158 — CHANGES_REQUESTED** [review]

File: TTBot/Models/ExcelSheetMappingModel.cs (hunk 1 above)
Comment (ruairigibney) [reply TTBot/Models/ExcelSheetMappingModel.cs:11]: Yeah 100% agree. At some point I'll create a command for adding new aliases and mappings to the DB from the bot

File: TTBot/Models/ChampionshipAliasesModel.cs (hunk 3 above)
Comment (ruairigibney) [reply TTBot/Models/ChampionshipAliasesModel.cs:12]: As per other comment.

File: TTBot/Program.cs (hunk 2 above)
Comment (ruairigibney) [reply TTBot/Program.cs]: Done

**Review by alex9158 — APPROVED** [review]

**Pull request merged**